nextflow.config

```diff
@@ -14,14 +14,29 @@ params {
   container = 'nfcore/openmspeptidequant:latest' // Container slug. Stable releases should specify release tag!
 
   help = false
-  reads = "data/*{1,2}.fastq.gz"
-  singleEnd = false
+  mzmls = "data/*.mzML"
+  fasta = "data/*.fasta"
   outdir = './results'
-  igenomes_base = "./iGenomes"
-  tracedir = "${params.outdir}/pipeline_info"
   clusterOptions = false
   awsqueue = false
   awsregion = 'eu-west-1'
+  num_threads = 5
+  fragment_mass_tolerance = 0.02
+  precursor_mass_tolerance = 5
+  fragment_bin_offset = 0
+  fdr_threshold = 0.01
+  fdr_level = 'peptide-level-fdrs'
+  number_mods = 3
+  num_hits = 1
+  digest_mass_range = "800:2500"
+  pick_ms_levels = 2
+  centroided = "False"
+  prec_charge = '2:3'
+  activation_method = 'ALL'
+  enzyme = 'unspecific cleavage'
+  fixed_mods = ''
+  variable_mods = 'Oxidation (M)'
+  tracedir = "${params.outdir}/pipeline_info"
 }
 
 profiles {
```

Comment:
We should add some profiles in here, ideally at least the `binac` and `cfc` ones. Have a look at nf-core/rrna-ampliseq to find out what you'd need to add here :-)